server/src/test/java/io/crate/integrationtests/PgTunnelLogicalReplicationITest.java

```diff
@@ -180,7 +180,7 @@ public void test_pg_client_can_use_password_authentication_for_replication() thr
     }
 
     @Test
-    public void test_cannot_create_subscription_with_invalid_user_password() throws Exception {
+    public void test_cannot_create_subscription_with_invalid_user_password_and_can_create_with_same_name_and_valid_credentials() throws Exception {
```

Comment:
Can we create a `RemoterClustersTest` unit test instead?

Reply:
done in https://github.com/crate/crate/pull/12507/commits/15c05ab199b54c27ec65f80d2e2e0c55d68c6053